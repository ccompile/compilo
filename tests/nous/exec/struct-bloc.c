
struct A
{
    int a;
    int b;
};

void print_obj(struct A* obj)
{
     putchar('0' + obj->a);
     putchar('0' + obj->b);
     putchar(10);
}

int main()
{
    {
	struct A obj;
	obj.a = 1;
	obj.b = 2;

	print_obj(&obj);
    }

    {
	struct A obj;

	print_obj(&obj);

	obj.a = 3;
	obj.b = 4;

	print_obj(&obj);
    }

    return 0;
}


