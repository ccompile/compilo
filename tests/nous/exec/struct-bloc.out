12
12
34
